tests/test_nn.py

```diff
@@ -645,3 +645,29 @@ def get_weights(m):
     conv = eqx.nn.Conv3d(5, 4, 3, key=getkey())
     conv = eqx.tree_at(lambda c: c.weight, conv, replace_fn=spectral)
     assert conv(jrandom.normal(getkey(), (5, 8, 8, 8))).shape == (4, 6, 6, 6)
+
+
+def test_pool(getkey):
+    Pool = eqx.nn.Pool("max", 2, (2, 2))
```

Comment:
As a python convention, variable names (here the `Pool` on the left hand side) should usually be in `snake_case`.